robot/impl/local_robot_test.go

```diff
@@ -3420,3 +3258,15 @@ func TestCloudMetadata(t *testing.T) {
 		})
 	})
 }
+
+//nolint:revive
+func initTestRobot(t *testing.T, ctx context.Context, cfg *config.Config, logger logging.Logger) (robot.LocalRobot, func()) {
```

Comment:
following [this proposed pattern for test helpers](https://sourcegraph.com/blog/go/advanced-testing-in-go#test-helpers)

> Never return errors. Just pass in *testing.T and fail.